server/src/main/java/org/elasticsearch/index/codec/tsdb/es819/ES819TSDBDocValuesProducer.java

```diff
@@ -1262,6 +1262,15 @@ public long longValue() throws IOException {
         }
     }
 
+    abstract static class BaseNumericDocValues extends NumericDocValues {
```

Comment:
I think we no longer need these base classes?

Reply:
Good point we no longer need these base classes. I will remove.

Reply:
Removed: 39dc98f93202e136a6097a9674a95fc385d6beb2